### DISCUSSION TITLE
Dotted line style for mesh plotting

### DISCUSSION THREAD

**Discussion by danpak94** [pyvista/pyvista#4501] [Q&A] [answered]
Hello,

Is it possible to use a dotted line (or any style other than solid lines) for plotting mesh edges?

Thank you

**Comment by banesullivan** [comment, marked as answer]
As far as I know this is not possible. But I want to encourage you to ask this question on the VTK discourse to see if I'm missing something: https://discourse.vtk.org/

**Reply by danpak94** [reply]
Thanks for the prompt response!
